src/utils.ts

```diff
@@ -7,3 +7,14 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPartItems(
+  items: string[],
+  currentPage: number,
+  pageSize: number,
+) {
+  const startIndex = (currentPage - 1) * pageSize;
```

Comment:
Consider checking that `currentPage` and `pageSize` are positive integers to avoid unexpected behavior when slicing the array. If they are not valid, you could return an empty array or throw an error.